xlml/apis/task.py

```diff
@@ -308,16 +308,16 @@ def clean_up(self, queued_resource: airflow.XComArg) -> DAGNode:
 
 
 @dataclasses.dataclass
-class TpuXpkTask(BaseTask):
-  """This is a class to set up tasks for TPU provisioned by XPK tool.
+class XpkTask(BaseTask):
+  """This is a class to set up tasks for TPU/GPU provisioned by XPK tool.
 
   Attributes:
-    task_test_config: Test configs to run on this TPU.
-    task_gcp_config: Runtime TPU creation parameters.
+    task_test_config: Test configs to run on this TPU/GPU.
+    task_gcp_config: Runtime TPU/GPU creation parameters.
     task_metric_config: Metric configs to process metrics.
   """
 
-  task_test_config: test_config.TestConfig[test_config.Tpu]
+  task_test_config: test_config.TestConfig[None]
```

Comment:
Will it work use `test_config.Accelerator`?

Reply:
Changing to Accelerator has the error `type 'Accelerator' is not subscriptable`